scenarioo-client/app/components/detailarea/detailarea.component.css

```diff
@@ -37,23 +40,25 @@ sc-detailarea .detail-row {
 
 sc-detailarea .detail-cell {
     display: table-cell;
-    /*visibility: hidden;*/
 }
 
 sc-detailarea .detail-button {
     background-color: #CCCCCC;
     width: 30px !important;
-    vertical-align: middle;
     font-size: x-large;
     color: #337AB7;
-    text-align: center;
     cursor: pointer;
 }
 
+sc-detailarea #arrowCenter {
```

Comment:
is an id (arrowCenter) really needed or could an additional class achieve the same?

Reply:
An additional class could achieve the same. 
But is it not better to use an id in this case? It appears only once...

Reply:
an id, is something that needs to be unique on the whole page. I would only use this in styling, if it is for sure something that will never never never occur twice on one page. So I would not use an id here, you never know, one day there could be two detail areas on the same page, couldnt it?

Reply:
See also https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4

Reply:
All right, thanks.